diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8dcceb4..b17ea28 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt	
@@ -30,9 +30,6 @@ if(MSVC)
 endif()
 # android
 # target_link_libraries(${PROJECT_NAME} -L/data/data/com.termux/files/usr/lib/libGLESv2.so.2)
-# its sole existence is to fuck around with the mod developer name
-
-target_compile_definitions(geode-sdk INTERFACE GEODE_EXPOSE_SECRET_INTERNALS_IN_HEADERS_DO_NOT_DEFINE_PLEASE)
 # include("${CMAKE_CURRENT_SOURCE_DIR}/src/devtools/CMakeLists.txt")
 # setupCustomDevtools()
 # Set up dependencies, resources, and link Geode.
